nano/scripts/nano/5_nano_base_callbacks.js

```diff
@@ -1,39 +1,42 @@
 // NanoBaseCallbacks is where the base callbacks (common to all templates) are stored
```

Comment:
This file has a shitload of spacing changes because it was half indented with tabs and half indented with spaces. Now it's indented with only spaces.